airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -0,0 +1,2668 @@
+# user's password, and raise a warning in the UI instead. Usually when you have that many sessions, it means
+# that there is something wrong with your deployment - for example you have an automated API call that
+# continuously creates new sessions. Such setup should be fixed by reusing sessions or by periodically
+# purging the old sessions by using `airflow db clean` command.
+MAX_NUM_DATABASE_USER_SESSIONS = 50000
+
+
+class FabAirflowSecurityManagerOverride(AirflowSecurityManagerV2):
+    """
+    This security manager overrides the default AirflowSecurityManager security manager.
+
+    This security manager is used only if the auth manager FabAuthManager is used. It defines everything in
+    the security manager that is needed for the FabAuthManager to work. Any operation specific to
+    the AirflowSecurityManager should be defined here instead of AirflowSecurityManager.
+
+    :param appbuilder: The appbuilder.
+    """
+
+    auth_view = None
+    """ The obj instance for authentication view """
+    registeruser_view = None
+    """ The obj instance for registering user view """
+    user_view = None
+    """ The obj instance for user view """
+
+    """ Models """
+    user_model = User
+    role_model = Role
+    action_model = Action
+    resource_model = Resource
+    permission_model = Permission
+
+    """ Views """
+    authdbview = AuthDBView
+    """ Override if you want your own Authentication DB view """
+    authldapview = AuthLDAPView
+    """ Override if you want your own Authentication LDAP view """
+    authoidview = AuthOIDView
+    """ Override if you want your own Authentication OID view """
+    authoauthview = AuthOAuthView
+    """ Override if you want your own Authentication OAuth view """
+    authremoteuserview = AuthRemoteUserView
+    """ Override if you want your own Authentication REMOTE_USER view """
+    registeruserdbview = RegisterUserDBView
+    """ Override if you want your own register user db view """
+    registeruseroidview = RegisterUserOIDView
+    """ Override if you want your own register user OpenID view """
+    registeruseroauthview = RegisterUserOAuthView
+    """ Override if you want your own register user OAuth view """
+    actionmodelview = ActionModelView
+    permissionmodelview = PermissionPairModelView
+    rolemodelview = CustomRoleModelView
+    registeruser_model = RegisterUser
+    registerusermodelview = RegisterUserModelView
+    resourcemodelview = ResourceModelView
+    userdbmodelview = CustomUserDBModelView
+    resetmypasswordview = CustomResetMyPasswordView
+    resetpasswordview = CustomResetPasswordView
+    userinfoeditview = CustomUserInfoEditView
+    userldapmodelview = CustomUserLDAPModelView
+    useroauthmodelview = CustomUserOAuthModelView
+    userremoteusermodelview = CustomUserRemoteUserModelView
+    useroidmodelview = CustomUserOIDModelView
+    userstatschartview = CustomUserStatsChartView
+
```

Comment:
In the following block: I see that you just used it how it was in the old file (with some corrections), but I feel like the comment should be above the assignment it is describing.  I'll leave it up to you though, I don't feel too strongly about it.